cli/internal/uuid/uuid.go

```diff
@@ -0,0 +1,21 @@
+package uuid
+
+import (
+	guuid "github.com/google/uuid"
+)
+
+// https://github.com/spf13/pflag/issues/236#issuecomment-931600452
+
+type UUID struct {
+	guuid.UUID
+}
+
+func (u *UUID) Set(p string) error {
+	var err error
+	u.UUID, err = guuid.Parse(p)
```

Comment:
```suggestion
	u.UUID, err = guuid.Parse(str)
```

Reply:
fixed in [9131595](https://github.com/cloudquery/cloudquery/pull/18878/commits/91315957e0bf0a6b0013d4c4b870bd9d1eb1f254)